src/utils/ipfs.ts

```diff
@@ -10,9 +10,12 @@ export async function fetchJsonCached<T>(
   return await cache.lazy<T>(`ipfs-${cid}`, () => fetchJson<T>(cid));
 }
 
+const cidRegex = /^(Qm[1-9A-HJ-NP-Za-km-z]{44}|baf[0-9A-Za-z]{50,})$/;
```

Comment:
added this in here otherwise the build fails because of some malformed ipfs hashes in optimism